src/packages/documents/documents/tracked-reference/repository/document-tracked-reference.repository.ts

```diff
@@ -10,29 +10,19 @@ export class UmbDocumentTrackedReferenceRepository extends UmbControllerBase {
 		this.#trackedReferenceSource = new UmbDocumentTrackedReferenceServerDataSource(this);
 	}
 
-	async requestTrackedReference(unique: string, skip = 0, take = 20, filterMustBeIsDependency = false) {
+	async requestTrackedReference(unique: string, skip = 0, take = 20) {
 		if (!unique) throw new Error(`unique is required`);
-		return this.#trackedReferenceSource.getTrackedReferenceById(unique, skip, take, filterMustBeIsDependency);
+		return this.#trackedReferenceSource.getTrackedReferenceById(unique, skip, take);
 	}
 
-	async requestTrackedReferenceDescendantsFromParentUnique(
-		parentUnique: string,
-		skip = 0,
-		take = 20,
-		filterMustBeIsDependency = false,
-	) {
+	async requestTrackedReferenceDescendantsFromParentUnique(parentUnique: string, skip = 0, take = 20) {
```

Comment:
It doesn't look like this method is used. I think we should remove it for simplicity. The same goes for the media repo + data source.